akka-remote-tests/src/test/resources/reference.conf

```diff
@@ -1,6 +1,5 @@
 akka {
   actor {
-    serialize-messages = on
-    warn-about-java-serializer-usage = off
+    serialize-messages = off # FIXME try to enable again?
```

Comment:
which messages aren't serializable?

Reply:
I have given up on this, or rather, I don't think there is any point in trying to support `serialize-messages`, but don't want to do that here. It's ticket #26802